src/lib/routing-table.js

```diff
@@ -35,6 +35,33 @@ class RoutingTable {
     return false
   }
 
+  /**
+   * Compute a curve quote's `appliesToPrefix`, which is the shortest prefix
+   * that uniquely matches the target.
```

Comment:
explain that our routing tables do support subledgers, but only if the more generic route has no curve. If the generic route had a curve, the curve of the subledger would not override it. Therefore, routes with curves can only be siblings, even though routes without curves can have 'child' routes which apply only to a subledger (longer prefix)